x-pack/plugins/apm/public/components/shared/span_icon/get_span_icon.ts

```diff
@@ -33,12 +32,14 @@ const defaultTypeIcons: { [key: string]: string } = {
   resource: globeIcon,
 };
 
-const typeIcons: { [key: string]: { [key: string]: string } } = {
+export const typeIcons: { [type: string]: { [subType: string]: string } } = {
   aws: {
-    servicename: awsIcon,
+    servicename: 'logoAWS',
```

Comment:
Maybe since you're repeating `logoAWS` and `logoAzure` multiple times, would be a good idea to create a constant for them.

```suggestion
    servicename: LOGO_AWS,
```

Reply:
Would it make sense to invert it like:
```suggestion
    servicename: {
      aws: LOGO_AWS
    }
```

So we'd call `<SpanIcon type="servicename" subtype="aws" />`

Reply:
I think the better solution is that EUI exports this as a constant, so it can be used like:

```
servicename: ICONS.AWS
```


I've created an issue for this in the eui repo https://github.com/elastic/eui/issues/4877